testing.go

```diff
@@ -51,17 +52,16 @@ func StartTest(t *testing.T) *Test {
 func (test *Test) End() {
 	if r := recover(); r != nil {
 		test.span.SetTag("test.status", "ERROR")
-		_ = GlobalAgent.Flush()
-		panic(r)
```

Comment:
We don't want to affect how the testing framework would work without out instrumentation. If without Scope the test execution panics, it should panic too with our instrumentation.

Reply:
Yeah I get it, so we asume that we can only have one exception per run. (Because currently the go agent doesn't have concept of `currentSpan` so inner panics aren't gonna be reported.

As side note, stopping all test execution because an exception in one test and forcing you to re-run all tests to check if there are more errors is weird, I guess is a Go thing. Maybe we can add later an "option" to disable this panic.

Reply:
done, panic() reverted.